src/dodal/common/enums.py

```diff
@@ -0,0 +1,31 @@
+from ophyd_async.core import StrictEnum
+
+
+class OnState(StrictEnum):
```

Comment:
Could: I'm not a huge fan of the `<First Member>State` naming pattern and I'm wondering about suggesting something like:

```suggestion
class OnOff(StrictEnum):
```

Ditto for the rest of them. I'm not sure about this though, opinions welcome.

Reply:
Sorry, I didn't see @DiamondJoseph's review, we seem to be thinking exact opposites so ignore me

Reply:
Okay, have discussed with @DiamondJoseph offline, I'll comment there